web/ui/dashboard/src/app/private/private.service.ts

```diff
@@ -378,6 +378,21 @@ export class PrivateService {
 		});
 	}
 
+    deleteProject(): Promise<HTTP_RESPONSE> {
```

Comment:
@Oluwadaminiola Why did you move this here?

Reply:
I wanted to deprecate the settings service file as it only contained only one service @horlah